app/models/article.rb

```diff
@@ -72,13 +72,14 @@ class Article < ApplicationRecord
   validates :video_source_url, url: { allow_blank: true, schemes: ["https"] }
 
   before_validation :evaluate_markdown, :create_slug
-  before_create :create_password
+  before_save :update_cached_user
```

Comment:
The new `Rails/ActiveRecordCallbacksOrder` sorts callbacks in models following the order Rails will actually execute them, see https://github.com/rubocop-hq/rubocop-rails/pull/285